lib/components/common/StyledTable.tsx

```diff
@@ -0,0 +1,111 @@
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import { vars } from '../../theme/variables';
+
+const { gray50, gray100, gray200 } = vars;
+
+function createData(
+    subject: string,
+    species: string,
+    strain: string,
+    sex: string,
+    age: string,
+    groups: string
+) {
+    return { subject, species, strain, sex, age, groups };
+}
+
+const columns = ['Subject', 'Species', 'Strain', 'Sex', 'Age', 'Groups']
+
+const rows = [
+    createData('sub1', 'Mouse', 'C57BL167', 'Male', '6 weeks', 'sharing'),
+    createData('sub1', 'Mouse', 'C57BL167', 'Male', '6 weeks', 'sharing'),
+    createData('sub7', 'Mouse', 'C57BL167', 'Female', '6 weeks', 'sharing'),
+    createData('sub7', 'Mouse', 'C57BL167', 'Female', '6 weeks', 'sharing'),
+    createData('sub2', 'Mouse', 'C57BL167', 'Male', '6 weeks', 'sharing'),
+];
+
+export const StyledTable = () => {
+
+    return (
+        <TableContainer 
+            component={Paper} 
+            elevation={0}
+            sx={{
+                maxWidth: 650,
+                border: `1px solid ${gray100}`,
+                borderBottom: 0
+            }}
+        >
+            <Table
+                sx={{
```

Comment:
@Aiga115 in figma column width height is 120/44 , but you are using different values for table body and table header